srcpkgs/gnome-control-center/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'gnome-control-center'
```

Comment:
Added missing depends, as reported by warnings when launching via terminal, and they do not take much storage space. Lack of of `colord` led to crash when opening `color` panel:
```
(gnome-control-center:18633): GLib-GIO-ERROR **: 16:09:18.397: Settings schema 'org.freedesktop.ColorHelper' is not installed
Trace/breakpoint trap
```
Also added `libhandy-devel` to `makedepends` and modified template to pass `check` phase.
```diff

--- template.orig	2020-05-02 18:46:33.181466125 +0530
+++ template	2020-05-02 18:44:14.902468270 +0530
@@ -1,6 +1,6 @@
 # Template file for 'gnome-control-center'
 pkgname=gnome-control-center
-version=3.36.1
+version=3.36.2
 revision=1
 build_style=meson
 build_helper="gir"
@@ -10,24 +10,35 @@
 makedepends="ModemManager-devel accountsservice-devel alsa-plugins
  colord-gtk-devel gnome-bluetooth-devel gnome-desktop-devel gnome-menus-devel
  gnome-online-accounts-devel gnome-settings-daemon-devel grilo-devel ibus-devel
- libSM-devel libXxf86misc-devel libgnomekbd-devel libgtop-devel
+ libSM-devel libXxf86misc-devel libgnomekbd-devel libgtop-devel libhandy-devel
  libpwquality-devel mit-krb5-devel network-manager-applet-devel polkit-devel
  pulseaudio-devel samba-devel tzdata upower-devel vala-devel clutter-gtk-devel
  libxml2-devel libX11-devel gtk+3-devel cups-devel libgudev-devel libwacom-devel
  clutter-devel libsecret-devel libcanberra-devel udisks2-devel gsound-devel
  $(vopt_if cheese cheese-devel)"
-depends="alsa-plugins-pulseaudio desktop-file-utils
+depends="alsa-plugins-pulseaudio desktop-file-utils upower colord cups-pk-helper
  gsettings-desktop-schemas hicolor-icon-theme iso-codes sound-theme-freedesktop
  cracklib"
+checkdepends="python3-dbusmock xorg-server-xvfb"
 short_desc="GNOME control center"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://gitlab.gnome.org/GNOME/gnome-control-center"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=f3e32d1cff7ec2fc42c2db2c7f61274d476d3a49aae10040f57217ee25d7c690
+checksum=e0a5eff0c6d6c22db0c54fd070460de874d367ef3f9072e1559b70f28b847817
 
 build_options="cheese"
 desc_option_cheese="Add support for adding user account images with your webcam"
+
+pre_check() {
+	# test-network-panel needs id generated by dbus-uuidgen
+	dbus-uuidgen --ensure=/etc/machine-id
+}
+
+post_check() {
+	rm -f /etc/machine-id
+}
+
 if [ -z "$CROSS_BUILD" ]; then
 	build_options_default+=" cheese"
 fi
```

Reply:
I wonder if this package needs `/etc/machine-id` to run?
If yes, I think we need an update to `base-files`

Reply:
> I wonder if this package needs `/etc/machine-id` to run?

If `/etc/machine-id` isn't present then it gets the id from `/var/lib/dbus/machine-id` which is the default location used by `dbus-uuidgen`. Check passes in both cases so I just tossed a coin to choose 😅.

Reply:
I'm in favour of having test environment not too different from runtime environment.
Please suggest a new change for this.

Reply:
> I'm in favour of having test environment not too different from runtime environment.
> Please suggest a new change for this.

I've updated the diff.

Reply:
I've added the changes.